sdk/table/azure-table/azure/table/_shared/table_client_base.py

```diff
@@ -0,0 +1,60 @@
+# -------------------------------------------------------------------------
```

Comment:
I think you should make the name of this file _table_client_base to make it private